client/log_client.go

```diff
@@ -36,43 +32,27 @@ import (
 type LogClient struct {
 	LogID  int64
 	client trillian.TrillianLogClient
-	*logVerifier
+	LogVerifier
```

Comment:
Is it usual for struct embeds to be placed down amongst the fields like this?
I'd expect to see it first in the definition.

Reply:
Or actually name it. Any reason for this not to be
```
type LogClient struct {
  ...
  verifier LogVerifier
  ...
}
```

Reply:
OK I see that later on you are taking advantage of the embed to shorten the 'dotted path' to fields and methods.
I still think you should move the embed up to the top of this struct though, and have a separating empty line, as it gets a bit lost as it is here.

Reply:
Embeds should indeed come first.